examples/storage-examples/gluster-examples/nginx_gluster_host/README.md

```diff
@@ -0,0 +1,189 @@
+## Example 1: Deploy NGINX application using manually mounted GlusterFS volume from Atomic RHEL Host
+---
+
+
+**Summary**: 
+
+At this point, all [environment assumptions](..) are met, and we have a Gluster Cluster running on separate hosts (gluster1.rhs and gluster2.rhs for this example).  We also installed and mounted the Gluster Volumes on our Atomic RHEL hosts (myVol1 and myVol2).  We will use the myVol1 in this example to show a simple pod that uses Distributed Storage.  To take advantage of the DFS we will create a fuse mount to the GlusterFS servers called /mnt/gvol1 which will then be used as our hostPath in our pod definition.  This could be utilized as potential method or even work around for containers that need Distributed Storage.
```

Comment:
Its convention that the mount reflect the Gluster volume name.  /mnt/myVol1 and /mnt/myVol2 instead of /mnt/gvol1